kore/src/Kore/Rewrite/Axiom/EvaluationStrategy.hs

```diff
@@ -28,7 +28,6 @@ import Data.EitherR (
  )
 import Data.Semigroup (
     Min (..),
-    Option (..),
```

Comment:
From the base library:

>Deprecated: will be removed in GHC 9.2; use [Maybe](https://hackage.haskell.org/package/base-4.15.1.0/docs/Data-Maybe.html#t:Maybe) instead.
>
>[Option](https://hackage.haskell.org/package/base-4.15.1.0/docs/Data-Semigroup.html#t:Option) is effectively [Maybe](https://hackage.haskell.org/package/base-4.15.1.0/docs/Data-Maybe.html#t:Maybe) with a better instance of [Monoid](https://hackage.haskell.org/package/base-4.15.1.0/docs/Data-Monoid.html#t:Monoid), built off of an underlying [Semigroup](https://hackage.haskell.org/package/base-4.15.1.0/docs/Data-Semigroup.html#t:Semigroup) instead of an underlying [Monoid](https://hackage.haskell.org/package/base-4.15.1.0/docs/Data-Monoid.html#t:Monoid).
>
>Ideally, this type would not exist at all and we would just fix the [Monoid](https://hackage.haskell.org/package/base-4.15.1.0/docs/Data-Monoid.html#t:Monoid) instance of [Maybe](https://hackage.haskell.org/package/base-4.15.1.0/docs/Data-Maybe.html#t:Maybe).
>
>In GHC 8.4 and higher, the [Monoid](https://hackage.haskell.org/package/base-4.15.1.0/docs/Data-Monoid.html#t:Monoid) instance for [Maybe](https://hackage.haskell.org/package/base-4.15.1.0/docs/Data-Maybe.html#t:Maybe) has been corrected to lift a [Semigroup](https://hackage.haskell.org/package/base-4.15.1.0/docs/Data-Semigroup.html#t:Semigroup) instance instead of a [Monoid](https://hackage.haskell.org/package/base-4.15.1.0/docs/Data-Monoid.html#t:Monoid) instance. Consequently, this type is no longer useful.